relay-config/src/config.rs

```diff
@@ -1262,6 +1262,43 @@ pub struct GeoIpConfig {
     path: Option<PathBuf>,
 }
 
+/// Cardinality Limiter configuration options.
+#[derive(Serialize, Deserialize, Debug)]
+pub struct CardinalityLimiter {
+    /// The number of seconds to apply the limit to.
+    ///
+    /// Defaults to: 1 hour.
+    pub window_seconds: u64,
+    /// A number between 1 and `window_seconds`. Since `window_seconds` is a
+    /// sliding window, configure what the granularity of that window is.
+    ///
+    /// If this is equal to `window_seconds`, the quota resets to 0 every
+    /// `window_seconds`.  If this is a very small number, the window slides
+    /// "more smoothly" at the expense of having much more redis keys.
+    ///
+    /// The number of redis keys required to enforce a quota is `window_seconds /
+    /// granularity_seconds`.
+    ///
+    /// Defaults to: 10 minutes.
```

Comment:
it seems that default is set to 60 seconds only.

Reply:
good catch, thanks, phew!